### DISCUSSION TITLE
Welcome to go-sqlcmd Discussions!

### DISCUSSION THREAD

**Discussion by shueybubbles** [microsoft/go-sqlcmd#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

With this project being so new, we'd love your input on how to prioritize future development. For example:
- What missing features from the old sqlcmd are most important to replicate in the new sqlcmd? 
- What challenges do you encounter trying to use the tools in cloud-native applications and workflows?

**Comment by subramn01** [comment]
I am trying to use sqlcmd utility now as I am moving from 20 years in Sybase to SqlServer. One thing I would very very very commonly use in the Interactive SQL prompt is to 'edit' the queries with 'vi' command. Now in both SqlCmd and go-sqlcmd, I try to find a similar function. I found there is one variable SQLCMDEDITOR to set your desirable editor, but I am in NO luck to make it to work. Very unfortunate!

**Reply by shueybubbles** [reply]
thx for the feedback! Adding the editor command to go-sqlcmd is on my to-do list. I hope to have it in the next few weeks.

**Comment by Jhenernovoa** [comment]
Me pueden indicar como se pasa el parametro de la contraseña en esta version?

**Reply by shueybubbles** [reply]
La contraseña es una variable de entorno
```
set SQLCMDPASSWORD=mypassword
sqlcmd -U username 
```

**Reply by Jhenernovoa** [reply]
Muchas Gracias!!! , no sabia como hacerle

**Reply by Jhenernovoa** [reply]
Estimado, disculpe que le moleste tanto, Ahora tengo el siguiente error
TLS Handshake failed: tls: server selected unsupported protocol version 301

Estoy intentando conectarme a SQL server 2008.

**Reply by Jhenernovoa** [reply]
ya encontre la solucion aca https://github.com/denisenkom/go-mssqldb/issues/726#issuecomment-1162755429

Gracias!!,,
